src/smartcontracts/typesystem/managedDecimal.ts

```diff
@@ -84,8 +89,8 @@ export class ManagedDecimalSignedValue extends TypedValue {
     private readonly value: BigNumber;
     private readonly scale: number;
 
-    constructor(value: BigNumber.Value, scale: number) {
-        super(new ManagedDecimalType(scale));
+    constructor(value: BigNumber.Value, scale: number, isVariableDecimals: boolean = false) {
```

Comment:
Parameter name `isVariableDecimals` is not consistent with parameter name of `ManagedDecimalValue` constructor. Maybe `isVariable` for both of them?